earthpy/__init__.py

```diff
@@ -1,4 +1,4 @@
-"""Utility functions for the earthlab class."""
+"""Utility functions for the working with spatial data."""
```

Comment:
This is the module-level docstring for all of earthpy, rather than the dostring for the `earthpy.io` function. If you have a better summary of all of the stuff that earthpy does, we should definitely put it here, but it should include the spatial operations, data getting, plotting, etc., as this is the high-level description. 

Do you have a preferred summary of all the things earthpy does @lwasser? If so, we can put that here!

Reply:
oh i totally missed this was in the `__init__.py` i thikn because it said utility functions for the earth lab class which i assumed was `earthLabData`. I think the language there is just fine if it's for the entire package and not just that class. We can edit it later too if need be. Thank you!! we can resolve this!